app/domain/authentication/authn_oidc/fetch_oidc_secrets.rb

```diff
@@ -0,0 +1,58 @@
+require 'command_class'
+require 'conjur/fetch_required_secrets'
+
+module Authentication
+  module AuthnOidc
+
+    # TODO: (later version) Fix CommandClass so we can add errors directly
+    # inside of it
+    #
+    # TODO: list any OIDC or other errors here.  The errors are part of the
+    # API.
+    #
+    # Errors from FetchRequiredSecrets
+    #
+    RequiredResourceMissing = ::Conjur::RequiredResourceMissing
+    RequiredSecretMissing   = ::Conjur::RequiredSecretMissing
+
+    FetchOidcSecrets = CommandClass.new(
+      dependencies: {
+        fetch_secrets: ::Conjur::FetchRequiredSecrets.new
+      },
+      inputs:       %i(conjur_account service_id required_variable_names)
```

Comment:
Why the space after `inputs:` here?

Reply:
just default indentation of RubyMine. Changed it now in preferences.